how-to/integrate-with-sso/integrate-with-auth0/README.md

```diff
@@ -54,7 +54,7 @@ There is also a second entry for `https://accounts.google.com` in the `logoutUrl
 If you enable additional federated providers in Auth0 you might need additional logout urls.
 
 3. Optional (if you wish to pin the version of OpenFin Workspace to version 10.0.0 and you are on Windows) - Set Windows registry key for [Desktop Owner Settings](https://developers.openfin.co/docs/desktop-owner-settings).
```

Comment:
This readme should update when you resync with main (which is now v11) but failing that it should say workspace 11.0.0 instead of 10.0.0 now.